package.json

```diff
@@ -32,6 +32,10 @@
     "url": "git://github.com/ramda/ramda.git"
   },
   "main": "index.js",
+  "files": [
+    "src",
+    "dist"
+  ],
```

Comment:
Ya, looks like it is included because of the 'main' field? Explicitly including it is fine with me. The files at the bottom of [this section](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) are also included.